### ISSUE TITLE
Basemap thumbnails link to localhost:9876 on embedded Builder Map on Mat-Su

### ISSUE THREAD

**Issue by jasonferrier** [trailheadlabs/outerspatial-map-library#166]
The basemap images on Mat-Su's site are attempting to load from `localhost:9876`.

Their map is embedded via iFrame to [https://www.outerspatial.com/builder_maps/21/fullscreen](https://www.outerspatial.com/builder_maps/21/fullscreen), so this seems like a bug with map-library and not how the embedded the map.

![mat-su_map-library_bug](https://user-images.githubusercontent.com/136632/43535425-337dea36-9577-11e8-9cf3-4075605e6acf.png)

**Comment by jasonferrier** [comment]
#162 would fix this issue if we want to punt it. 🏈

**Comment by jasonferrier** [comment]
We can probably just have some type of flag like on `team-website` for the [NAC config]:(https://github.com/trailheadlabs/team-website/blob/e4798de51b2c3023bdd8e4f95de2e8e08178241f/labs/projects/nac/assets/js/app-new.js#L4)

```
var mapLibraryPath = development ? 'http://localhost:9876/' : 'https://staging-cdn.outerspatial.com/libs/outerspatial-map-library/2018-07-31-18:13/';
```

**Comment by jasonferrier** [comment]
This is also affecting Manager in production. It can be seen in the map builder section.

**Comment by jasonferrier** [comment]
It looks like something to do with this line of code in [switcher.js:96](https://github.com/trailheadlabs/outerspatial-map-library/blob/master/src/control/switcher.js#L96)
```
img.src = window.L.Icon.Default.imagePath + '/control/switcher/' + imgFileName + '.png';
```

**Comment by jasonferrier** [comment]
It seems as though Map Library being loaded into Manager was version 7.1.11 [(details in other issue comment)](https://github.com/trailheadlabs/outerspatial-manager/issues/401#issuecomment-412701485) and when I updated the path to 7.1.16, the actual code loaded was 7.1.15. Nate and I are thinking that what may have happened was that the deploy script was run without running another build and the 7.1.15 code was pushed into the 7.1.16 S3 directory.

Idea to fix would be to rebuild and re-deploy the 7.1.16 release to 7.1.16.1 since we won't be able to redeploy/overwrite to the existing directory.

Nate is taking a look at fixing the build failure right now that seems to have been caused by 61d02e3a3502e48f2155b2e5d41c85a5175ea261
